hub/experimental/dataloader.py

```diff
@@ -52,13 +52,16 @@ def __init__(
     def batch(self, batch_size: int, drop_last: bool = False):
         """Returns a batched DataLoader object.
```

Comment:
Also, I guess we should mention `Hub3DataLoader` object since there is no "DataLoader" object per se